hudi-flink-datasource/hudi-flink1.14.x/src/main/java/org/apache/hudi/adapter/Utils.java

```
@@ -57,4 +64,8 @@ public static BinaryExternalSorter getBinaryExternalSorter(
     return new BinaryExternalSorter(owner, memoryManager, reservedMemorySize,
         ioManager, inputSerializer, serializer, normalizedKeyComputer, comparator, conf);
   }
+
+  public static InternalSchema applyTableChange(InternalSchema oldSchema, List changes, Function<LogicalType, Type> convertFunc) {
+    throw new RuntimeException("There is no possible to hit this method!");
```

Can we just throw a assertionError with simple msg ‘unexpected’